Added the option to paste text in shell

cychu42 opened the pull request:
I hope this is what you are looking for. Let me know if you need some changes.

## Description
Resovles #17 
The user would be able to add text into the shell for the tool to read, as long as the text is in `""` or `''`.
Usage: 
`metaxa read <text> <delay>`

Example: 
`metaxa read "Hello, my name is John. How are you?" 40`

## Changes
In index.js, a new option 'read' is added, keeping almost the same code as `start` option, minus the code for reading a file from a path.

makoteq commented:
Hey! , sorry fo the delay, solution is looking great so far, here are some issues that i have:

- Provide usage guide for introduced command inside "usage" const (we can make it in another PR if you like to)
- Swap places of`<text>` and `<delay>` making it look like this:
`metaxa read <delay> <text>`

after addresing those issues i will be happy to merge 🔥

cychu42 commented:
@makoteq 
No problem. I added the usage guide. I also added the if construct to achieve the second bullet point, without touching variable names.
This feature will ignore extra trailing input into the command, like: 
the `40` in `metaxa read "Hello, my name is John. How are you?" 40` 
the `somenonesense` in `metaxa read 60 "Hello, my name is John. How are you?" somenonesense`

makoteq commented:
Great work on this one 🚀
Thanks for contributing

cychu42 commented:
No problem! Thanks.
